src/includes/enriching-events/set-transaction-name/android.mdx

```diff
@@ -0,0 +1,3 @@
+```java
```

Comment:
We're trying to go Kotlin first for Android. Can we have the tabbed option with Kotlin, first?